python/nano/setup.py

```diff
@@ -72,7 +72,8 @@ def setup_package():
 
     pytorch_requires = ["torch==1.11.0",
                         "torchvision==0.12.0",
-                        "pytorch_lightning==1.4.2",
+                        "pytorch_lightning==1.6.4",
+                        "ray==1.12.1",
```

Comment:
ray is an optional dependency so you should not add it to `pytorch_requires`

If Ray is required for tests to pass, maybe there are some ray related code not moved to `nano/deps/ray`